tests/unit/test-class-wc-payments.php

```diff
@@ -117,51 +119,6 @@ public function test_rest_endpoints_validate_nonce() {
 		$this->assertEquals( 'woocommerce_rest_missing_nonce', $response->get_data()['code'] );
 	}
 
-	public function test_ajax_init_woopay_sends_correct_customer_id() {
-		// Necessary in order to prevent die from being called.
-		define( 'DOING_AJAX', true );
-
-		$customer_id = 'cus_123456789';
-
-		$pre_http_dispatch = function ( $result, $server, $request ) {
-			if ( in_array( $request->get_route(), [ '/wc/store/v1/checkout', '/wc/store/v1/cart' ], true ) ) {
-				return new WP_REST_Response( [ 'body' => wp_json_encode( [] ) ], 200 );
-			}
-
-			return $result;
-		};
-
-		$pre_http_request_cb = function ( $preempt, $parsed_args, $url ) use ( $customer_id ) {
-			$body = json_decode( $parsed_args['body'] );
-			$this->assertEquals( $customer_id, $body->customer_id );
-			return [ 'body' => wp_json_encode( [] ) ];
-		};
```

Comment:
This assertion is never called because the request on `Automattic/Jetpack/Connection/Client::remote_request` will always fail (it cannot be signed inside the test). This is the reason for the removal.